metrics-reporter-hive/src/main/java/org/apache/kylin/metrics/lib/impl/hive/HiveProducer.java

```diff
@@ -110,6 +114,8 @@ public void onRemoval(
             hostName = "UNKNOWN";
         }
         contentFilePrefix = hostName + "-" + System.currentTimeMillis() + "-part-";
+        supportAppend = !fs.getUri().toString().contains("s3");
```

Comment:
Not good.

Reply:
Check https://issues.apache.org/jira/browse/KYLIN-4427 , maybe `wasb://` should be added too, I am waiting for user's reply.